Configuration/Sets/WallsIoProxy/config.yaml

```
@@ -0,0 +1,2 @@
+name: jweiland/walls-io-proxy-default
```

We use "default" only, if we have multiple plugins where one plugin relates to data from another setting. maps2.
```suggestion
name: jweiland/walls-io-proxy
```